core/SConscript.kernel

```diff
@@ -296,6 +297,7 @@ ALLPATHS = [
         'embed/util/bl_check/inc',
         'embed/util/image/inc',
         'embed/util/rsod/inc',
+        'embed/util/scm_revision/inc',
```

Comment:
not needed in kernel anymore

Reply:
https://github.com/trezor/trezor-firmware/commit/883f54f367646bb095164447746da9e1a3c9e039